docs/pipeline/triggering/PERFORMANCE_TRIGGERS.md

```diff
@@ -0,0 +1,180 @@
+# Performance Triggering
+
+## Overview
+
+The `PerformanceTrigger` is a trigger mechanism that evaluates the performance of a model over time and makes decisions based on performance-observing criteria. Unlike simple triggers, the `PerformanceTrigger` considers user selected performance metrics and can evaluate decisions based on static or dynamic performance thresholds, as well as the number of avoidable misclassifications.
+
+We define `avoidable misclassifications` as the misclassifications that could have been avoided if the model had triggered earlier. They can be see as the lost benefits of not triggering. We derive the avoidable misclassification estimates by observing two time series:
+
+1. The series of evaluation scores directly after the last executed triggers (with fresh models). We define those results as `expected performance`.
+
+2. The series of evaluation SINCE the last trigger on policy update points where we could have triggered but decided not to. This series is used to estimate the avoidable misclassifications. We define those results as `actual/effective performance`.
+
+The delta between the expected and the actual performance is then used to estimate the avoidable misclassifications.
+
+This mechanism relies on performance and data density tracking to observe trends. Decision policies then make binary decisions (trigger or not) based on this information.
```

Comment:
data density occurs here the first time. i think this is still confusing me a bit because i don't know exactly what it is in this context and how you use it.